doc/content/manuals/include/fault_rupture/fault_rupture-theory.md

```diff
@@ -39,7 +39,7 @@ M_{zz}(t) &= M_o(t) \sin 2\delta \sin \lambda
 
 Each component of the above matrix is a force couple with the first index representing the force
 direction and the second index representing the direction in which the forces are separated (see
-[fig:source_direction]; [citet:aki2012quantitative]).
+[fig:source_direction]; [!citet](aki2012quantitative)).
```

Comment:
Its not working here for some reason..

Reply:
You found a super tricky regex problem, I will see if I can solve it but for now just use the suggestion.

Reply:
```suggestion
[fig:source_direction]; 
[!citet](aki2012quantitative)).
```